vault/identity_store_oidc_provider.go

```diff
@@ -238,7 +238,7 @@ func (i *IdentityStore) pathOIDCCreateUpdateProvider(ctx context.Context, req *l
 	if issuerRaw, ok := d.GetOk("issuer"); ok {
 		provider.Issuer = issuerRaw.(string)
 	} else if req.Operation == logical.CreateOperation {
-		provider.Issuer = d.Get("issuer").(string)
+		provider.Issuer = d.GetDefaultOrZero("issuer").(string)
```

Comment:
We should update all instances of this Get call (allowed_client_ids, scopes)

Reply:
Sorry about the misunderstanding! I think I got all instances of the above kind

Reply:
No problem!